app/src/main/res/layout/activity_detail.xml

```diff
@@ -51,6 +51,7 @@
                 android:id="@+id/toolbar"
                 android:layout_width="match_parent"
                 android:layout_height="?attr/actionBarSize"
+                android:fitsSystemWindows="true"
```

Comment:
@steffandroid really??I had removed this line , but performs badly....do you have a try?